servicecontrol/troubleshooting.md

```diff
@@ -105,3 +105,11 @@ The internal *FailedMessageRetries* collection must be purged in order to restor
 1. Select the "FailedMessageRetries" collection in the left tree
 1. Delete all documents in the collection
 1. Stop maintenance mode
+
+## Microsoft Defender SmartScreen blocks installer 
+
+All code is code signed, but Microsoft SmartScreen (part of Windows Defender) can considers the certificate "untrusted" and shows a warning when trying to install.
+
+The files are code signed correctly and show "NServiceBus Ltd" as the signee, but Microsoft needs to "build trust in" the certificate before not showing the warning. One of the main inputs is to get users to override the warning and install the files. It is required to override that warning by clicking "Run Anyway" until the NServiceBus code signing certificate gets trusted by Microsoft SmartScreen.
+
+All artifacts during builds are virus scanned before packaging.
```

Comment:
```suggestion

All artifacts during builds are virus scanned before packaging.
```

This doesn't seem to belong to this trouble shooting section.

Reply:
I read the rationale to put it here something like this:

Bad news, sorry that we forced this upon you. Here is how to resolve it. Trust us it is safe.

By the way our builds are trustworthy and yes you can really click on the button we just told you